File: NuGenVizDOC/NuGenVizDoc File x86/frmCapture.vb


Public Class frmCapture

    Private Sub frmCapture_FormClosed(ByVal sender As Object, ByVal e As System.Windows.Forms.FormClosedEventArgs) Handles Me.FormClosed
        Try
            Me.nuGenScreenCap1.Dispose()
            GC.Collect()
        Catch
        End Try
    End Sub
End Class

File: NuGenVizDOC/NuGenVizDoc DB ACCESS/SCreenCap_03.10.06/Capture/frmCapture.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace Capture
{
    public partial class frmCapture : Form
    {
        private Form parentForm;

        public Form ParentForm
        {
            get
            {
                return parentForm;
            }
            set
            {
                parentForm = value;
            }
        }

        public frmCapture()
        {
            InitializeComponent();
        }

        private void frmCapture_VisibleChanged(object sender, EventArgs e)
        {
            
        }

        private void frmCapture_FormClosed(object sender, FormClosedEventArgs e)
        {
            this.parentForm.Show();
        }
    }
}